src/__test__/SearchPage.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import SearchPage from '../pages/SearchPage';
+
+const mockStore = configureStore([]);
+
+describe('SearchPage component', () => {
+  test('SearchPage component renders correctly', () => {
+    const initialState = {
+      city: {
+        value: [
+          {
+            name: 'City Name',
+            lat: 11.1111,
+            lon: 22.2222,
+          },
+
+        ],
+      },
+    };
+
+    const store = mockStore(initialState);
+    const tree = renderer.create(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={['/search/City Name']}>
```

Comment:
- [ ] Good job adding the tests as requested by the previous reviewer. However, this test is failing. Please make sure to fix it and all tests should pass without any errors.

![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/83803414/d1dd7387-ece9-4a30-8e15-579ce315c77e)